pkg/reconciler/apis/apiexport/apiexport_controller.go

```diff
@@ -172,11 +173,11 @@ func NewController(
 type controller struct {
 	queue workqueue.RateLimitingInterface
 
-	kcpClusterClient kcpclient.ClusterInterface
+	kcpClusterClient kcpclient.Interface
```

Comment:
now that the ClusterInterface type goes away we need convention even more about the variable names. Do we agree that we always call them `*ClusterClient` and never ever `*Client`, and not even `*Clients`?

Reply:
I'm fine with having it `ClusterClient` everywhere, that's why didn't change the naming convention which we had earlier and kept it as is to avoid confusion.